projects/kaltura-ng/kaltura-ui/src/lib/dynamic-form/dynamic-form-control-base.ts

```diff
@@ -25,6 +26,7 @@ export abstract class DynamicFormControlBase<T>{
     order: number;
     description: string;
     validators : Array<ValidatorFn>;
+    validateOn?: 'change' | 'blur' | 'submit';
     errors:  {[key: string]:  string};
     inputHelperConfig: {title?: string, body: string};
     styleClass: string;
```

Comment:
@tan-tan-kanarek - missing initialization of validateOn property in the constructor